index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portofolio: setup and mobile version skeleton</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body class="">
+    <header class="navibar">
+      <nav class="nav-container">
+        <a class="logo" href="#">My Logo</a>
+        <img class="mobile-menu" src="img/Union.png" alt=" navigation" />
+      </nav>
+    </header>
+    <section class="h-container">
+      <div class="container">
+        <h1 class="first-text">I’m Karamaga <br />Glade to see you!</h1>
```

Comment:
- [ ] As pointed out by the previous reviewer, please let's work once more on the width of this header. If you take a look keenly at the Figma design, you will notice that the width of the paragraph beneath this header is larger. Seems you did but the opposite. Let's make sure that we stick to Figma design as much as possible.

Your Implementation | Figma Design
:---:|:---:
![](https://user-images.githubusercontent.com/63087491/159741202-915e9548-67ea-494e-9e06-570a0542908e.png) | ![](https://user-images.githubusercontent.com/63087491/159741945-eca41f9f-6ac7-42bd-82ca-0dd4468f3e18.png)